### ISSUE TITLE
[AU4] Effects - Remove wxWidgets dependency from AU3 effects 

### ISSUE THREAD

**Issue by ianalarina** [audacity/audacity#7089]
## Description

Currently, all Audacity effects have UI dependency on wxWidgets. We need to get rid of it in order to migrate them on Qt framework.

#### Rework needed (blocked by design)

- [x] Equalization, replacing
     - [x] Graphic EQ
     - [x] Filter Curve EQ
- [x] Noise Reduction ~~(with new UI/flow)~~ can be postpone

#### "P0" (UI-less effects which probably don't need to depend on any UI toolkit):

- [x] Fade In #7167
- [x] Fade Out #7167
- [x] Invert #7167
- [x] Reverse #7167
- [x] Stereo To Mono #7167
- [x] Repair #7167

#### P1 (common effect, no special UI)

- [x] Amplify #7091
- [x] Bass and Treble #7169
- [x] Normalize #7169
- [x] Loudness Normalization#7169
- [x] Echo #7169
- [x] Reverb #7169
- [x] Change Pitch #7169 (~~use SP stretcher by default here~~ later)
- [x] Change Tempo #7169 (~~use SP stretcher by default here~~ later)
- [x] Change Speed and Pitch #7169
- [x] Nyquist plugins #7180

#### P2 (less common effect and/or special UI)
- [x] Auto Duck
- [x] Distortion
- [x] Click Removal
- [x] Compressor #7017
- [x] Limiter #7017
- [x] Phaser
- [x] Repeat
- [x] Sliding Stretch
- [x] Truncate Silence
- [x] Wahwah

#### P3 (niche and legacy effects)

- [x] Paulstretch
- [x] ScienFilter (Classic Filters)
- [x] Legacy compressor

Analyzers, tools and generators are P3 as well, with the exception of Silence which is P2.

### Analyzers
- [x] Contrast
- [x] Find Clipping
- [x] Plot Spectrum

### Generators
- [x] Chirp #7094
- [x] DTMF Tones
- [x] Noise
- [x] Silence (P2)
- [x] Tone #7094

### Tools
- [x] Nyquist Prompt

**Comment by saintmatthieu** [comment]
@LWinterberg if you could add priority values (I suggest 1 to 3) to each item? That might be helpful.

**Comment by LWinterberg** [comment]
I'll look into details more when I'm back, but I separated out a couple of UI-less effects which just immediately apply to the audio. You may want to do those first.

There also is the question of a realtime-capable EQ with a new UI. @DilsonsPickles has a design concept for that somewhere, which might not be realizable with wx. So EQ probably is blocked by design atm.
